### PR TITLE
[k8s-dashboard] Increase mem limit and remove cpu limit.

### PR DESCRIPTION
**Pull request by ydye**

### CODE REVIEW

**Comment by fanyangCS** [comment]
I suggest we enforce cpu limits as well.

File: pai-management/k8sPaiLibrary/template/dashboard-deployment.yaml.template [hunk 1]
```diff
@@ -46,11 +46,9 @@ spec:
         resources:
           # keep request = limit to keep this container in guaranteed class
           limits:
-            cpu: 100m
```
[the remark this answers is not in the document]
Comment (fanyangCS) [reply pai-management/k8sPaiLibrary/template/dashboard-deployment.yaml.template]: Please add a comment: the ideal cpu setting will be 3.5, according to our experiment. If the server hosting k8s dashboard has enough resource, user can change this setting to a larger value.

[the remark this answers is not in the document]
Comment (ydye) [reply pai-management/k8sPaiLibrary/template/dashboard-deployment.yaml.template]: Done

[the remark this answers is not in the document]
Comment (fanyangCS) [reply pai-management/k8sPaiLibrary/template/dashboard-deployment.yaml.template]: how about adding some comment?

---
In reply to: [205312694](https://github.com/Microsoft/pai/pull/990#discussion_r205312694) [](ancestors = 205312694)

[the remark this answers is not in the document]
Comment (ydye) [reply pai-management/k8sPaiLibrary/template/dashboard-deployment.yaml.template]: Done

**Pull request merged**